packages/trpc/server/routers/viewer/bookings.tsx

```diff
@@ -472,6 +479,30 @@ export const bookingsRouter = router({
           },
         });
 
+        // delete scheduled jobs of cancelled bookings
+        cancelScheduledJobs(bookingToReschedule);
```

Comment:
cancel scheduledJobs and all workflow reminders for the booking if rescheudling is requested